.taskcluster.yml

```diff
@@ -35,20 +35,20 @@ tasks:
         taskId: ${taskGroupId}
         taskGroupId: ${taskGroupId}
         schedulerId: taskcluster-github
-        provisionerId: aws-provisioner-v1
-        workerType: github-worker
+        provisionerId: proj-git-cinnabar
+        workerType: ci
         retries: 5
         scopes:
           $flatten:
             - assume:repo:github.com/${e.base_login}/${e.base_repo_name}:decision-task
             - $if: 'tasks_for == "github-push"'
               then:
-              - 'queue:route:index.github.${e.base_login}.${e.base_repo_name}.*'
-              - secrets:get:repo:github.com/${e.base_login}.${e.base_repo_name}:codecov
+              - 'queue:route:index.project.git-cinnabar.${e.base_login}.*'
```

Comment:
The reason the index key contained the login before is that IIRC that's the prefix I had permission to write to (which is fair, considering how the prefix was). But now that git-cinnabar appears first, and considering only pushes to the canonical repo are supposed to be indexed, I feel like the login part should go away here.